EIPS/eip-5604.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: 5604
+title: NFT Lien
```

Comment:
```suggestion
title: NFT Liens
```

Reply:
Thank you for the suggetion.

We prefer to use the singular form to refer to a type of thing. Such as 
- Lien (instead of Liens) as in https://en.wikipedia.org/wiki/Lien
- Token (instead of Tokens) as in eip-20.md

Reply:
Fair enough.